public/app/features/integrated-alerting/IntegratedAlertingPage.tsx

```diff
@@ -1,7 +1,53 @@
-import React, { FC } from 'react';
+import React, { FC, useMemo, useState } from 'react';
+import { TabsBar, TabContent, Tab, useStyles } from '@grafana/ui';
+import { Messages } from './IntegratedAlerting.messages';
+import { getStyles } from './IntegratedAlerting.styles';
+import { TabKeys } from './IntegratedAlerting.types';
 
-const IntegratedAlertingPage: FC = ({ children }) => {
-  return <>{children}</>;
+const IntegratedAlertingPage: FC = () => {
+  const styles = useStyles(getStyles);
+  const [activeTab, setActiveTab] = useState(TabKeys.alerts);
+  const tabs = useMemo(
```

Comment:
Do we really need `useMemo` here?

Reply:
I think yes, so it doesn't declare the tabs on every render. It will make even more sense once we start to add the components inside each tab.